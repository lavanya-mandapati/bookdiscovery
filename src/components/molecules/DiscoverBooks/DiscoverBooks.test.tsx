import React from "react";
import { render} from "@testing-library/react";
import BookDiscover from './DiscoverBooksImage'


describe("Topics renders correctly",()=>
{
    test("should match shapshot",()=>{
    const{asFragment}=render(<BookDiscover />);
    expect(asFragment()).toMatchSnapshot()

    })
})