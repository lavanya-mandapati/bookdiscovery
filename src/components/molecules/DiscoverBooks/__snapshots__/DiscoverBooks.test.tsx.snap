// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topics renders correctly should match shapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-container-1"
  >
    <img
      alt="Snow"
      src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/296A1994-087A-4F86-BC94-D8E3D96946EE.png"
    />
    <button
      class="makeStyles-btn-2"
    >
      Discover
    </button>
  </div>
</DocumentFragment>
`;
